packages/server/build.gradle

```diff
@@ -111,14 +111,6 @@ tasks.named('test') {
     useJUnitPlatform()
 }
 
-hibernate {
-    enhancement {
-        lazyInitialization true
-        dirtyTracking true
```

Comment:
This brief article is as clear as mud. https://dev.to/yigi/hibernate-dirty-check-4j6m#:~:text=When%20an%20entity%20object%20is,called%20%E2%80%9CHibernate%20Dirty%20Check%E2%80%9D.

Reply:
Yeah my understanding is that if you make changes to an object in a transaction, it will actually write them out. Honestly Id rather that _not_ be the case. I don't want changes pushed to the db unless i explicitly want to write them.